week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -1,64 +1,122 @@
+/*
+ In this exercise you will deal with Objects stored in array and DOM manipulation with JavaScript.
+ From now, exercise descriptions don't provide detailed examples about input and output of functions.
+ You should be already able to read and understand the code in order to extract the necessary information
+ you need to implement functions' body.
+ */
+
+/*
+ Optional question:
+
+ Notice how this JavaScript file linked to the index.html file.
+ What do you think when this JavaScript file is being executed?
+
+ When you finished this exercise try to move the script tag around the Div tag with the id "content",
+ refresh the page and observe what happens. (This should answer the question above.)
+ 
+ What do you think why this changes shouldn't work?
+ */
+// LEAVE YOUR ANSWER HERE (THIS IS OPTIONAL)
+
 /**
+ * This function recieves a lists of people. Each object should contain the name and the occupation of a person.
+ * Look for usage of the function in the code and see what variable is passed into it as an argument.
  *
- * For each of the names in the array passed into this function
- *
- * - Add a <h1> tag to the website containing the name of the person
- * - Add a <h2> tag to the website containing the job of the person
+ * Create and insert the below HTML elements to the DOM for each of Objects in the Array as follows:
+ *  1. Add a <h1> tag to the website containing the name of the person
+ *  2. Add a <h2> tag to the website containing the job of the person
  *
- * All of your HTML should go inside the Div tag with the id "content".
+ * All of your HTML elements should go inside the Div tag with the id "content".
  *
+ * An example "content" div should look like:
  * <div id="content">
- *      <h1>{Name Here}</h1>
- *      <h2>{Job Here}</h2>
- *      .....
+ *      <h1>Mario</h1>
+ *      <h2>Plumber</h2>
+ *      <h1>Luigi</h1>
+ *      <h2>Plumber</h2>
  * </div>
  */
-function exerciseOne(arrayOfPeople) {
+function insertPeopleData(arrayOfPeople) {
   let content = document.querySelector("#content");
+  //Write your code in here
+  for (let i = 0; i < arrayOfPeople.length; i++) {
+    var h1Tag = document.createElement("h1");
```

Comment:
Try to avoid `var` - stick to `let` and `const` wherever possible. I will default to using `const` unless it becomes clear that I will need to reassign the variable. 😄